mla/test_statistics.py

```diff
@@ -52,13 +51,15 @@ def __post_init__(self) -> None:
             sob_spatial:
 
         Raises:
-            ValueError: There must be at least one event.
+            RuntimeWarning: There should be at least one event.
         """
 
         self.n_events = len(self.events)
 
         if self.n_events == 0:
-            return None
+            warnings.warn(''.join(['You are trying to preprocess zero events. ',
```

Comment:
It will still throw out warnings if the time window is too short and no background ends out in a trial. I am thinking maybe  suppressing that warning in produce_and_minimize preprocessing step(line 140 in core.py)

Reply:
I was under the impression that we were going to be scaling to different time windows after the fact. In either case, it's easy to suppress warnings